core/spec/models/spree/variant_spec.rb

```diff
@@ -520,4 +520,28 @@
       expect(variant.dimension).to eq (dimension_expected)
     end
   end
+
+  context '#discontinue!' do
+    let(:variant) { create(:variant) }
+
+    it "sets the discontinued" do
+      variant.discontinue!
+      variant.reload
+      expect(variant.discontinued?).to be(true)
+    end
+  end
+
+  context "#discontinued?" do
+    let(:variant_live) { build(:variant) }
+    it "should be false" do
+
+      expect(variant_live.discontinued?).to be(false)
+    end
+
+    let(:variant_discontinued) { build(:variant,
+                                       discontinue_on: Time.now - 1.day)  }
```

Comment:
Expression at 542, 75 should be on its own line.<br>Unnecessary spacing detected.